README.md

```diff
@@ -1,48 +1,55 @@
-# openEO Project Steering Committee Guidelines
+# openEO Project Steering Committee
 
-Author: Edzer Pebesma, mostly copied from https://gdal.org/development/rfc/rfc1_pmc.html#rfc-1
+This document describes how the openEO Project Steering Committee determines membership, and makes decisions on openEO project issues. The openEO project is concerned with all repositories (software and specifications etc.) found under https://github.com/Open-EO.
+
+In brief the committee votes on proposals on https://github.com/Open-EO/PSC/issues. Proposals are available for review for at least two business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
 
-Contact: edzer.pebesma@uni-muenster.de
+## Members
 
-Status: Proposed
+Matthias Mohr is declared initial Chair of the openEO Project Steering Committee. The initial list of members of the Project Steering Committee is:
 
-## Summary
+* Jeroen Dries (@jdries)
+* Luca Foresta (@lforesta)
+* Alexander Jacob (@aljacob)
+* Miha Kadunc (@mkadunc)
+* Matthias Mohr (**chair**, @m-mohr)
+* Markus Neteler (@neteler)
+* Edzer Pebesma (@edzer)
 
-This document describes how the openEO Project Steering Committee determines membership, and makes decisions on openEO project issues. The openEO project is concerned with all software found under https://github.com/Open-EO.  
-In brief the committee votes on proposals on https://github.com/Open-EO/PSC. Proposals are available for review for at least two (German) business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
+The current list will be maintained by the openEO chair on this page and will be published to the openEO web pages, too.
 
 ## Detailed Process
 
-1. Proposals are written up and submitted on https://github.com/Open-EO/PSC for discussion and voting, by any interested party, not just committee members.
-2. Proposals need to be available for review for at least two business days before a final decision can be made.
-3. Respondents may vote “+1” to indicate support for the proposal and a willingness to support implementation.
-4. Respondents may vote “-1” to veto a proposal, but must provide clear reasoning and alternate approaches to resolving the problem within the two days.
-5. A vote of -0 indicates mild disagreement, but has no effect. A 0 indicates no opinion. A +0 indicates mild support, but has no effect.
+1. Proposals are written up and submitted on https://github.com/Open-EO/PSC/issues/new for discussion and voting, by any interested party, not just committee members.
+2. Proposals need to be available for review for at least two business days before a final decision can be made. Public holidays count as business days.
```

Comment:
Beforehand it was "(German) business days". That is still ambiguous due to different public holidays in different federal states (e.g. NRW). So I'd just say we ignore business days at all. As there are sometimes two public holidays in a row (e.g. Christmas), we may want to decide that "at least two business days" is changed to "at least three business days". Opionions?